src/libPMacc/include/particles/ParticlesBase.kernel

```diff
@@ -636,56 +636,119 @@ struct KernelFillGaps
     }
 };
 
+/** delete all particles within an area
+ *
```

Comment:
I will add a warning `@warning the particle memory of the particle is not byte wise zeroed` because I will avoid do describe how it is implemented because the current way to check if a particle in a frame is valid is by testing the `multiMask` but if we change this behavior the comment will be outdated.

One way to refactor could be #793